### PR TITLE
feat: Bump keyborg to 2.0.0 and tabster to 4.1.1

### LINKED ISSUE #26124: [Bug]: Menu trigger does not have focus indicator if menu is opened by click and closed by Esc
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
https://react.fluentui.dev/?path=/docs/components-menu-menu--default
```

### Are you reporting Accessibility issue?

no

### Reproduction

https://react.fluentui.dev/?path=/docs/components-menu-menu--default

### Bug Description

Click on menu trigger to open the menu
Press ESC to close the menu

## Actual Behavior
Focus is on trigger but the focus indicator is not visible

## Expected Behavior
Focus indicator should be visible as the focus was moved using keyboard

### Logs

_No response_

### Requested priority

High

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### LINKED ISSUE #26459: Home/End keys should navigate to first/last element in the row
### Cell focus mode

According to guidance from the APG https://www.w3.org/WAI/ARIA/apg/example-index/grid/dataGrids

### Row focus mode

Home/End should focus on first/last rows

### LINKED ISSUE #26460: Handle PgDn and PgUp correctly for virtualized and unvirtualized
The keys should use default behaviour in Table/DataGrid. However when virtualized those keys should scroll the virtualized container.

This should be done by preventing the default behaviour of PgDn PgUp in tabster

### LINKED ISSUE #26534: [Bug]: [Popover] Modalizer must have either aria-label or aria-labelledby
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (16) x64 AMD Ryzen 7 5800X 8-Core Processor
    Memory: 17.96 GB / 31.91 GB
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (102.0.1245.41)
    Internet Explorer: 11.0.19041.1566
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codesandbox.io/s/bold-neco-fuin5l

## Actual Behavior
When open the popover, I get console log of this warning. 

image

It can be fixed by adding aria-label / aria-labelledby to popover surface. But still wondering if this is an accessibility issue. 

## Expected Behavior
Expect to not have this warning;

### Logs

_No response_

### Requested priority

Normal

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by ling1726**
See title

Fixes #26459
Fixes #26460
Fixes #26124
Fixes #26534

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                        |        Size (minified/GZIP) |
| ------------------------------------------------------------------------------------------------------------------------ | --------------------------: |
| <samp>priority-overflow</samp> <br /> <abbr title='bundle-size/overflow-manager.fixture.js'>createOverflowManager</abbr> |  `3.182 kB`<br />`1.308 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>   |          `69 B`<br />`89 B` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>             | `30.908 kB`<br />`6.714 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>           | `17.776 kB`<br />`5.141 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr>                  |        `180 B`<br />`159 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/40fdbb373ad339586df18ce6afc43434210c159b'>40fdbb373ad339586df18ce6afc43434210c159b</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9424"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/BaseButton-mount_base.html">1182</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/BaseButton-mount.html">1202</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Breadcrumb-mount_base.html">2798</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Breadcrumb-mount.html">2867</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Checkbox-mount_base.html">2630</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Checkbox-mount.html">2653</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/CheckboxBase-mount_base.html">2360</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/CheckboxBase-mount.html">2375</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/ChoiceGroup-mount_base.html">4309</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/ChoiceGroup-mount.html">4280</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/ComboBox-mount_base.html">1194</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/ComboBox-mount.html">1196</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/CommandBar-mount_base.html">9287</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/CommandBar-mount.html">9391</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/ContextualMenu-mount_base.html">10270</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/ContextualMenu-mount.html">10301</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/DefaultButton-mount_base.html">1372</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/DefaultButton-mount.html">1377</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/DetailsRow-mount_base.html">3273</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/DetailsRow-mount.html">3262</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/DetailsRowFast-mount_base.html">3271</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/DetailsRowFast-mount.html">3276</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/DetailsRowNoStyles-mount_base.html">3168</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/DetailsRowNoStyles-mount.html">3218</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Dialog-mount_base.html">2928</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Dialog-mount.html">2936</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/DocumentCardTitle-mount_base.html">586</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/DocumentCardTitle-mount.html">588</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Dropdown-mount_base.html">3160</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Dropdown-mount.html">3164</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/FocusTrapZone-mount_base.html">1940</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/FocusTrapZone-mount.html">1961</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/FocusZone-mount_base.html">1985</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/FocusZone-mount.html">1970</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/GroupedList-mount_base.html">1808</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/GroupedList-mount.html">2014</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/GroupedList-virtual-rerender_base.html">1098</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/GroupedList-virtual-rerender.html">1082</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">1605</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/GroupedList-virtual-rerender-with-unmount.html">1603</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/GroupedListV2-mount_base.html">551</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/GroupedListV2-mount.html">563</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/GroupedListV2-virtual-rerender_base.html">538</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/GroupedListV2-virtual-rerender.html">552</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">550</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">555</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/IconButton-mount_base.html">1784</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/IconButton-mount.html">1780</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Label-mount_base.html">747</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Label-mount.html">754</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Layer-mount_base.html">4183</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Layer-mount.html">4190</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Link-mount_base.html">853</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Link-mount.html">848</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/MenuButton-mount_base.html">1626</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/MenuButton-mount.html">1608</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/MessageBar-mount_base.html">2406</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/MessageBar-mount.html">2322</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Nav-mount_base.html">3069</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Nav-mount.html">3068</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/OverflowSet-mount_base.html">1409</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/OverflowSet-mount.html">1415</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Panel-mount_base.html">2509</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Panel-mount.html">2516</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Persona-mount_base.html">1298</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Persona-mount.html">1291</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Pivot-mount_base.html">1528</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Pivot-mount.html">1513</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/PrimaryButton-mount_base.html">1491</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/PrimaryButton-mount.html">1477</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Rating-mount_base.html">6951</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Rating-mount.html">6947</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/SearchBox-mount_base.html">1470</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/SearchBox-mount.html">1499</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Shimmer-mount_base.html">2912</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Shimmer-mount.html">3002</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Slider-mount_base.html">2109</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Slider-mount.html">2144</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/SpinButton-mount_base.html">4273</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/SpinButton-mount.html">4304</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Spinner-mount_base.html">822</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Spinner-mount.html">838</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/SplitButton-mount_base.html">2846</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/SplitButton-mount.html">2850</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Stack-mount_base.html">864</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Stack-mount.html">858</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/StackWithIntrinsicChildren-mount_base.html">2231</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/StackWithIntrinsicChildren-mount.html">2267</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/StackWithTextChildren-mount_base.html">4956</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/StackWithTextChildren-mount.html">4953</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/SwatchColorPicker-mount_base.html">9461</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/SwatchColorPicker-mount.html">9479</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/TagPicker-mount_base.html">2327</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/TagPicker-mount.html">2347</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/TeachingBubble-mount_base.html">76877</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/TeachingBubble-mount.html">75336</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Text-mount_base.html">822</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Text-mount.html">816</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/TextField-mount_base.html">1556</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/TextField-mount.html">1567</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/ThemeProvider-mount_base.html">1448</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/ThemeProvider-mount.html">1441</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/ThemeProvider-virtual-rerender_base.html">1137</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/ThemeProvider-virtual-rerender.html">1139</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1995</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1992</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Toggle-mount_base.html">1133</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/Toggle-mount.html">1124</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/buttonNative-mount_base.html">530</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26584/perf-test/buttonNative-mount.html">540</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

File: change/@fluentui-react-tabster-b2d43df9-1387-475e-b4e3-a4204207aba2.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-tabster-b2d43df9-1387-475e-b4e3-a4204207aba2.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26584&commitId=04e18f724db5475260197bfe884748522049967f&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

File: packages/react-components/react-tabster/etc/react-tabster.api.md [hunk 2]
```diff
@@ -68,8 +68,8 @@ export const useFocusFinders: () => {
     findAllFocusable: (container: HTMLElement, acceptCondition?: ((el: HTMLElement) => boolean) | undefined) => HTMLElement[];
     findFirstFocusable: (container: HTMLElement) => HTMLElement | null | undefined;
     findLastFocusable: (container: HTMLElement) => HTMLElement | null | undefined;
-    findNextFocusable: (currentElement: HTMLElement, options?: Pick<Types.FindNextProps, 'container'>) => HTMLElement | null | undefined;
-    findPrevFocusable: (currentElement: HTMLElement, options?: Pick<Types.FindNextProps, 'container'>) => HTMLElement | null | undefined;
+    findNextFocusable: (currentElement: HTMLElement, options?: Pick<Partial<Types.FindNextProps>, 'container'>) => HTMLElement | null | undefined;
```
[the remark this answers is not in the document]
Comment (ling1726) [reply packages/react-components/react-tabster/etc/react-tabster.api.md:71]: Yeah the `container` option is required now

**Review by Hotell — APPROVED** [review]

File: change/@fluentui-react-tabster-b2d43df9-1387-475e-b4e3-a4204207aba2.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-react-tabster-b2d43df9-1387-475e-b4e3-a4204207aba2.json:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26584&commitId=faf9c56a43ac909cb97ad452faa444ecf2f47f04&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1828 screenshots unchanged, 0 screenshots with different dimensions and 4 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoButton.default - Dark Mode.active.chromium.png|626| Changed|
|InfoButton.default - High Contrast.active.chromium.png|1141| Changed|
|InfoButton.default.active.chromium.png|1126| Changed|
|InfoButton.size.rest.chromium.png|1195| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

Comment (fabricteam) [inline change/@fluentui-react-tabster-b2d43df9-1387-475e-b4e3-a4204207aba2.json:1]: ## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26584&commitId=faf9c56a43ac909cb97ad452faa444ecf2f47f04&env=prod)

✅ There was 1 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.Image Example Avatar.chromium.png|0| Added|

</details>


<div id="vrtCommentFluentUI-v0"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

File: change/@fluentui-react-tabster-b2d43df9-1387-475e-b4e3-a4204207aba2.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-react-tabster-b2d43df9-1387-475e-b4e3-a4204207aba2.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26584&commitId=6306ae51549a9199b1cee4dfe4066664f6da7613&env=prod)

✅ There was 1 screenshots added, 0 screenshots removed, 1044 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Persona.Persona with children.default.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Pull request merged**